packages/cozy-client/src/assets/qualifications.json

```diff
@@ -144,13 +144,13 @@
       "label": "other_work_document"
     },
     {
-      "label": "health_book",
-      "purpose": "report",
+      "label": "health_certificate",
+      "purpose": "attestation",
       "sourceCategory": "health",
-      "subjects": ["history"]
+      "subjects": ["identity", "capacity", "vaccine"]
     },
     {
-      "label": "national_insurance_card",
+      "label": "national_healthInsurance_card",
```

Comment:
I would prefer `national_health_insurance_card` to cope with the the camel_case nomenclature used in this file ;)